src/Features/Core/Portable/CodeStyle/NotificationOption.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal class NotificationOption
```

Comment:
for WPF binding.. If i use the enum, i need an ObjectDataProvider to retrieve values from `Enum.GetValues` method. whereas by making a type and having properties and ToString() on the type, I'm making it easier for data binding.